src/actions/components.js

```diff
@@ -8,7 +8,13 @@ const componentStart = (component) => ({
   payload: component
 });
 
+const componentRemove = (component) => ({
+  type: 'COMPONENT_REMOVE',
+  payload: component
+});
+
 export {
   componentEnd,
-  componentStart
+  componentStart,
```

Comment:
Nit: alphabetize?

Reply:
if we do that here, would I alphabetize the function order in this file and the order of switch case statements in reducers/components.js?

Reply:
Yeah, makes sense to do it in both places.